yang/frr-isisd.yang

```diff
@@ -881,6 +881,68 @@ module frr-isisd {
           description
             "This leaf describes the state of the interface.";
         }
+
```

Comment:
Put the 'container adjacency-sids' definition under a new grouping called 'adjacency-sids' outside and have just the 'uses adjacency-sids' statement here.